### PR TITLE
Update links to project and packages

### PR DESCRIPTION
**Pull request by sethaxen**
- Update link to arviz.org
- Update links to packages

### CODE REVIEW

**Comment by sethaxen** [comment]
@OriolAbril there's one last reference here:
https://github.com/arviz-devs/arviz-project/blob/e68391541a38074befe27bcdd55016f5a370d901/conf.py#L57
Can we remove that one too?

**Comment by OriolAbril** [comment]
>  Can we remove that one too?

yes, that one should be updated too, thanks! :heart:

**Comment by sethaxen** [comment]
Done! I assumed it actually should point to the project site instead of Python ArviZ's site.

File: conf.py [hunk 1]
```diff
@@ -54,4 +54,4 @@
     "dark_logo": "ArviZ_white.png",
 }
 
-intersphinx_mapping = {"arviz": ("https://arviz-devs.github.io/arviz/", None)}
+intersphinx_mapping = {"arviz": ("https://www.arviz.org/", None)}
```
Comment (OriolAbril) [inline conf.py:57]: This should continue to be the python one, it is used to ease linking to the python docs and make the references more robust. 

We have for example a link to the contributing page in the python docs (and to the julia one) in https://github.com/arviz-devs/arviz-project/blob/main/our_team.md. The link to python one uses this intersphinx feature, now updating this, that and any other links to the python docs are updated simply by rebuilding the website.

```suggestion
intersphinx_mapping = {"arviz": ("https://python.arviz.org/en/latest/", None)}
```

Comment (sethaxen) [reply conf.py:57]: Aaah, okay, thanks for the explanation. My sphinx is rusty!

Comment (OriolAbril) [reply conf.py]: It should work for non-sphinx generated docs as long as there is an `objects.inv` file with the right format indicating the labels and their respective url

**Comment by OriolAbril** [comment]
Checked the links before merging and everything was working now :), thanks for updating everything

**Pull request merged**